browser/brave_shields/ad_block_service_browsertest.cc

```diff
@@ -1781,6 +1781,32 @@ IN_PROC_BROWSER_TEST_F(AdBlockServiceTest, RemoveparamTopLevelNavigation) {
   ASSERT_TRUE(dynamic_server_.ShutdownAndWaitUntilComplete());
 }
 
+// `$removeparam` should not be activated in default blocking mode
+IN_PROC_BROWSER_TEST_F(AdBlockServiceTest, DefaultNoRemoveparam) {
```

Comment:
is there an existing test that checks that it is activated in aggressive mode?

Reply:
it seems that the test sets aggressive mode by default which is very confusing because aggressive mode is not the default. Can you please change this in a follow-up? Calling `DisableAggressiveMode()` in tests to get the default behavior is really not great imo. You should consider splitting this test up by content setting/feature and also possibly by content setting value like `AdBlockServiceCosmeticFilteringStandardModeTest`